### PR TITLE
[feat][fix][refactor] Several modifications

### PR DESCRIPTION
**Pull request by tuzi3040**
Plz check commit for details.

### CODE REVIEW

**Review by zyzsdy — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -5,12 +5,11 @@
 
 ## Dependencies
 
-* openssl
 * mkvmerge (optional, mkv output required)
 
-! Minyami requires Node 10.5.0+
+! Minyami requires Node 10.22.0+
```
[the remark this answers is not in the document]
Comment (tuzi3040) [reply readme.md:10]: Raise node version requirment to 10.22.0 which upgrades openssl sources to 1.1.1g.

File: src/utils/media.ts [hunk 2]
```diff
@@ -170,7 +171,10 @@ export async function decrypt(
     key: string,
     iv: string
 ) {
-    return await exec(
-        `openssl aes-128-cbc -d -in "${input}" -out "${output}" -K "${key}" -iv "${iv}"`
-    );
+    const algorithm = 'aes-128-cbc';
+    const decipher = crypto.createDecipheriv(algorithm, Buffer.from(key, 'hex'), Buffer.from(iv, 'hex'));
```
[the remark this answers is not in the document]
Comment (tuzi3040) [reply src/utils/media.ts]: Should be fixed in 171b3e93d19dc8bf5ae8f141e5405373fadcbd02.

File: src/utils/media.ts [hunk 3]
```diff
@@ -172,7 +172,21 @@ export async function decrypt(
...
+        iv = '0' + iv;
+    }
+    const keyBuffer = Buffer.alloc(16);
+    const ivBuffer = Buffer.alloc(16);
+    keyBuffer.write(key, 'hex');
+    ivBuffer.write(iv, 16 - iv.length / 2, 'hex');
+
+    const decipher = crypto.createDecipheriv(algorithm, keyBuffer, ivBuffer);
     const i = fs.createReadStream(input);
     const o = fs.createWriteStream(output);
```
[the remark this answers is not in the document]
Comment (tuzi3040) [reply src/utils/media.ts:191]: It seems ok

**Comment by Last-Order** [comment]
LGTM, thank you.

**Comment by Last-Order** [comment]
Landed in 3.1.8

**Pull request merged**